transformer/auto.go

```diff
@@ -49,6 +49,12 @@ func init() {
 		Alloc:   func() interface{} { return &Data{} },
 		Help:    "Enforces custom-rules for data fields of metrics.",
 	})
+	Auto.Add(skogul.Module{
+		Name:    "parse",
+		Aliases: []string{},
+		Alloc:   func() interface{} { return &Parse{} },
+		Help:    "Parses a metric using a skogul parser",
```

Comment:
I would perhaps add something about it being somewhat recursive. Something like "Re-parse a field in a metric using a Skogul parser. E.g.: If a metric is json-encoded, but a field in the metric is a text-string of key=value pairs, the handler will parse the JSON, and this parse transformer can be used to parse the key=value pairs." ?